packages/components/src/components/Breadcrumbs/Breadcrumb.stories.tsx

```diff
@@ -38,11 +95,61 @@ export const Default: Story = {
 };
 
 export const NoRoot: Story = {
+  render: (args) => (
+    <Breadcrumbs {...args}>
+      {(item) => (
+        <Breadcrumb id={item['@id']} href={item['@id']}>
+          {item.title}
+        </Breadcrumb>
+      )}
+    </Breadcrumbs>
+  ),
+  args: {
+    items: [
+      { '@id': '/folder', title: 'Folder' },
+      { '@id': '/folder/page', title: 'Page' },
+    ],
+  },
+};
+
+export const SlotWithRootWithIcons: Story = {
+  render: (args) => <BreadcrumbsSlot {...args} />,
+  args: {
+    root: { '@id': '/', title: 'Home', icon: <HomeIcon size="sm" /> },
+    items: [
+      { '@id': '/folder', title: 'Folder', icon: <FolderIcon size="sm" /> },
+      { '@id': '/folder/page', title: 'Page', icon: <PageIcon size="sm" /> },
+    ],
+  },
+};
+
+export const LotsOfItems: Story = {
+  render: (args) => (
+    <Breadcrumbs {...args}>
+      {(item) => (
+        <Breadcrumb id={item['@id']} href={item['@id']}>
+          {item.title}
+        </Breadcrumb>
+      )}
+    </Breadcrumbs>
+  ),
   args: {
-    root: '/',
-    includeRoot: false,
+    root: { '@id': '/', title: 'Home', icon: <HomeIcon size="sm" /> },
     items: [
       { '@id': '/folder', title: 'Folder' },
+      { '@id': '/folder/folderB', title: 'Folder with long name' },
+      {
+        '@id': '/folder/folderB/folderC',
+        title: 'Folder with long name and a bit more',
+      },
+      {
+        '@id': '/folder/folderB/folderC/folderD',
+        title: 'Folder with long name even more long',
```

Comment:
This one should be longer than the previous one, not the same length.

```suggestion
        title: 'Folder with the longest folder name ever, please make it stop, aaaaaaaahhhhhh!',
```

Reply:
I don't know if it matters. Maybe it has different behavior at different break points when you have a super long name. Right now you can't test a super long name because the lengths of the strings are the same.

Reply:
Yeah, and I also just thought of translations, and what would it look like with German words, such as `Neunmilliardeneinhundertzweiundneunzigmillionensechshunderteinunddreißigtausendsiebenhundertsiebzigfache`.

Reply:
@stevepiercy FYI, this is just one flavour of the breadcrumbs implementation, not the main one, in case that you need it. We are going away of the "one component does it all" way of doing things. Also subject to continue improving it over time.